include/curl/curl.h

```diff
@@ -2378,6 +2378,9 @@ typedef enum {
 CURL_EXTERN CURLsslset curl_global_sslset(curl_sslbackend id, const char *name,
                                           const curl_ssl_backend ***avail);
 
+/* Get the selected SSL backend. */
+CURL_EXTERN const curl_ssl_backend *curl_global_sslget();
```

Comment:
GCC and Clang complain because there is no `void` between those parentheses. I am fairly certain that things will work nicely if this line is changed to

```c
CURL_EXTERN const curl_ssl_backend *curl_global_sslget(void);
```